pypeit/par/pypeitpar.py

```diff
@@ -944,6 +944,10 @@ def __init__(self, spectrograph=None, pipeline=None, detnum=None, sortroot=None,
         descr['qadir'] = 'Directory relative to calling directory to write quality ' \
                          'assessment files.'
 
+        defaults['redux_path'] = ''
```

Comment:
Should we change this to cwd or something?

Reply:
yup, done